src/core/applications/services/pedidoService.ts

```diff
@@ -38,14 +40,16 @@ export default class PedidoService {
       );
     }
 
+    const pagamento = await this.checkoutRepository.geraPagamento(metodoDePagamentoId, pedido);
     const fatura = await this.faturaRepository.criaFatura({
       pedidoId,
       metodoDePagamentoId,
+      qrCode: pagamento.qrCode,
```

Comment:
Só teremos QR se o método de pagamento for QRCode, certo?

Eu to achando que Checkout e Fatura é meio que redundante 🤔

Reply:
```Só teremos QR se o método de pagamento for QRCode, certo?```
Sim

```Eu to achando que Checkout e Fatura é meio que redundante thinking```
Sim, checkout retorna a fatura

Reply:
não seria o caso talvez de adicionar `fatura` no fluxo e no dicionário? pra diferenciar `fatura` de `pedido`. acham que faz sentido?

Reply:
Acho que seria bom adicionar essa definição no dicionário. No domain story telling acho que ele não chega a entrar, talvez no Event Storming. 🤔

Reply:
estou olhando o event storming no miro mas não estou muito certa de que momento ele entraria